templates/privacy/public.tpl

```diff
@@ -0,0 +1,64 @@
+    some information you'd rather keep private. We value that privacy and wish to assure you that we have taken steps to
+    ensure that by requesting an account here, you are not at risk of exposing your identity on the internet. At the
+    same time, however, in order to properly process your account request, we do need to collect certain information
+    that will allow us to distinguish you from others editing from the same or a nearby location.</p>
+
+<h3>Wikimedia Labs Disclaimer</h3>
+
+<p>By using this project, you agree that any private information you give to this project may be made publicly available
+    and not be treated as confidential.</p>
+<p>By using this project, you agree that the volunteer administrators of this project will have access to any data you
+    submit. This can include your IP address, your username/password combination for accounts created in Labs services,
+    and any other information that you send. The volunteer administrators of this project are bound by the Wikimedia
+    Labs Terms of Use, and are not allowed to share this information or use it in any non-approved way.</p>
+<p>Since access to this information is fundamental to the operation of Wikimedia Labs, these terms regarding use of your
+    data expressly override the Wikimedia Foundation's Privacy Policy as it relates to the use and access of your
+    personal information.</p>
+
+<h3>What data do we collect, and why?</h3>
+<p>This system records your IP Address and useragent data. This information may be processed overseas and your data will
+    remain confidential. It is important to note that this information is provided by your internet browser to any
+    website you visit; it is not possible to confirm any specific person's identity with this information. We only use
+    this to prevent abuse of Wikipedia and ACC.
+    We also require your email address so that we can respond to you with questions and the result of your request.
+    By submitting an account request at ACC, you agree to provide this information and allow ACC volunteers to view it
+    for the explicit purpose of reviewing your account request on the English Wikipedia.</p>
+
+<h3>How is this data stored, and who can see it?</h3>
+<p>We store this data in a secure database, which is visible only to ACC developers, several of whom are identified to
+    the Wikimedia Foundation, just as Checkusers are required to do. In order to assist with reviewing your block, this
+    information is provided to ACC volunteers as follows:</p>
+<ul>
+    <li>Status update emails are sent through the system, but to prevent abuse and to be able to correlate you emailing
+        us, your email is visible all volunteers.</li>
+    <li>Your useragent will only be visible to ACC Senior Developers and those reviewing the block that have access to
+        the CheckUser tool on Wikipedia. CheckUsers need to see your useragent to help differentiate you from the person
```

Comment:
This one I wish to leave in place. If need be we can link to the CU page on wikipedia, but it defines a specific group of people, and not people who can do x task. At worst, we could also directly link to the Special:ListUsers checkuser group.